firmware/src/main.c

```diff
@@ -54,6 +54,7 @@ int main(void)
 	fpga_io_init();
 	led_init();
 	debug_spi_init();
+	allow_fpga_takeover_usb(true);
```

Comment:
What if the PROGRAM button is pressed and Apollo takes over the USB port in the next block of code? Next time the FPGA is configured, Apollo will hand off the port as soon as it receives advertisement messages. No need to call the `allow_fpga_takeover_usb` vendor request from the host. Is that the desired behavior?

Reply:
If PROGRAM is pressed at start-up, `force_fpga_offline()` will call `allow_fpga_takeover_usb(false)`. I placed `allow_fpga_takeover_usb(true)` above the button handling specifically so the override in `force_fpga_offline()` would work.